cmd/contour/serve.go

```diff
@@ -267,6 +259,9 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 		return fmt.Errorf("error parsing request timeout: %w", err)
 	}
 
+	// TLS 1.1 is still allowed, but it's not the default for HTTPProxies/Ingresses.
+	minimumTLSVersion := annotation.MinTLSVersion(ctx.TLSConfig.MinimumProtocolVersion, envoy_api_v2_auth.TlsParameters_TLSv1_1)
```

Comment:
Should this set the min version to TLSV1_2?

Reply:
We/I have gone back and forth on this a bit in https://github.com/projectcontour/contour/issues/2777, but I was currently planning to leave this value at 1.1 for at least another release before changing its default, to give users some more time to address this situation if needed. Leaving this setting at 1.1 allows proxies that explicitly specify 1.1 to continue working as-is.

Reply:
Reading this I thought it was setting 1.1 to the default. Where does 1.2 become the default? Somewhere else in the builder?

Reply:
Yeah it's in the processors (e.g. https://github.com/projectcontour/contour/pull/3007/files#diff-203dc8cde1099ac3666b0b37414f448aR184-R185).

This setting acts as a global minimum, so if any proxy or ingress has a min TLS version lower than this value, then it will be (silently) upgraded to this global minimum when programming Envoy. I can add some more comments to explain that behavior.